soem/ethercatcoe.h

```diff
@@ -78,8 +78,8 @@ void ecx_SDOerror(ecx_contextt *context, uint16 Slave, uint16 Index, uint8 SubId
 int ecx_SDOread(ecx_contextt *context, uint16 slave, uint16 index, uint8 subindex,
                       boolean CA, int *psize, void *p, int timeout);
 int ecx_SDOwrite(ecx_contextt *context, uint16 Slave, uint16 Index, uint8 SubIndex,
-    boolean CA, int psize, void *p, int Timeout);
-int ecx_RxPDO(ecx_contextt *context, uint16 Slave, uint16 RxPDOnumber , int psize, void *p);
+                 boolean CA, int psize, const void *p, int Timeout);
```

Comment:
If you don't mind, can you take the opportunity to indent the existing long function wrappings in the same manner in this header file. I know it is not uniformly throughout SOEM but let’s try to make it the same in single files.
Make the same change as you did for 'ecx_SDOwrite'

ec_SDOread
ec_SDOwrite
ecx_SDOread
ecx_SDOwrite

Reply:
done